blueprints/ember-cli-changelog/files/config/release.js

```diff
@@ -14,8 +14,8 @@ module.exports = {
   // format: 'YYYY-MM-DD',
   // timezone: 'America/Los_Angeles',
 
-  beforeCommit: function(project, versions) {
+  beforeCommit: function(project, tags) {
     console.log(project);
-    return generateChangelog.call(project);
+    return generateChangelog(project, tags.latest, tags.next);
```

Comment:
I updated this line so that it would reflect my other changes, but like I said before I'd like to avoid making the user add this to their `config/release.js`, so I think the issue is moot.